src/Sia.Data.Incident/Filters/UncorrelatedEventFilters.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Sia.Data.Incidents.Models;
+using Sia.Shared.Data;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Sia.Data.Incidents.Filters
+{
+    public class UncorrelatedEventFilters:Filters<Event>
```

Comment:
It seems reasonable to reuse EventFilters rather than create another Filter class; you can just validate in the controller action that incidentId isn't provided

Reply:
I do not know how to distinguish these two situation.